tf2/include/tf2/time.h

```diff
@@ -58,10 +58,9 @@ namespace tf2
 
   inline std::string displayTimePoint(const TimePoint& stamp)
   {
-    std::time_t time = std::chrono::system_clock::to_time_t(std::chrono::time_point_cast<std::chrono::milliseconds>(stamp));
-    char str[100];
-    std::strftime(str, sizeof(str), "%c", std::localtime(&time));
-    return std::string(str);
+    char buffer [100];
```

Comment:
I first thought it was using `%d` and then the buffer could be smaller. But its actually `%f` which mean 100 characters might not be enough depending on the value. I don't think that a hard coded value will do for `%f` a double. Also in case it does get truncated I think this function need to signal the error case.